Content.Server/Ame/Components/AmeControllerComponent.cs

```diff
@@ -72,10 +72,23 @@ public sealed partial class AmeControllerComponent : SharedAmeControllerComponen
     [DataField("nextUpdate")]
     public TimeSpan NextUpdate = default!;
 
+    /// <summary>
+    /// The next time this will try to update the controller UI.
```

Comment:
Instead of doing this, why not make all the AME UI's update at the same time? I know that there are other systems that do per-component updates, but because there is likely going to be at most 2 AME's on any given map, this doesn't really matter.

Reply:
I just copied the mechanism behind the AmeController update. It is easily understandable and lightweight in processing time since it only uses addition and simple compares. 

The only example i found where all components of a type are updated at the same time was the PowerMonitoringConsole. But copying that solution over lead to code with more lines and two different update-checking processes that each iterate over all components at some point. So I don't see any benefit here without also changing how the normal controller updates are handled, which i see as out of scope for this PR.

In the I agree with you in the regard that with the low amount of AME controllers, it really does not matter how they determine when to update their UIs as long as they do so on a regular base. tl;dr: I'll keep this as it is